CHANGELOG.md

```diff
@@ -50,7 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
```

Comment:
For the getSessionWithoutModifyingResponse function, if it return an invalid claim error, the err object from that response can directly be sent to the frontend as a JSON serialisable with 403? Or does something else need to be done